components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+import data from "../../data/notifications.json";
+
+const Banner = ({ type, description, date, isOpen }: any) => {
+  const [isDismissed, setIsDismissed] = React.useState(true);
+
+  React.useEffect(() => {
+    const datesArray = JSON.parse(localStorage.getItem("notifications")) || [];
+
+    // se a notificação não foi dismissed
+    if (!datesArray.includes(date)) setIsDismissed(false);
+  }, [date]);
+
+  function handleDismiss(dismissed: boolean) {
+    setIsDismissed(dismissed);
+
+    // guarda a data da notificação no localStorage
+    let datesArray = JSON.parse(localStorage.getItem("notifications")) || [];
+    datesArray.push(date);
+    localStorage.setItem("notifications", JSON.stringify(datesArray));
+  }
+
+  return (
+    <div
+      className={`${
+        isDismissed || isOpen ? "translate-y-full" : ""
```

Comment:
Also, just a small trick

Whenever you have a binary condition (If-Then) you can use `&&` operator, instead of the ternary (If-Then-Else) operator. It is a cleaner way to write this styling conditions.

so you can change `isDismissed || isOpen ? "translate-y-full" : ""` to `(isDismissed || isOpen) && "translate-y-full"`

Reply:
nice, thx ;)